app/sidero-controller-manager/internal/ipxe/ipxe_test.go

```diff
@@ -25,5 +25,5 @@ func TestEmbeddedLength(t *testing.T) {
 	}))
 
 	// iPXE script should fit length of the reserved space in the iPXE binary
-	assert.Less(t, buf.Len(), 1000)
+	assert.Less(t, buf.Len(), 1015)
```

Comment:
let's not bump it.. we can drop comments.. either in the code, or filter them out when generating the final script (with proper unit-tests for it)